src/vaillant/15.basv3.tsp

```diff
@@ -0,0 +1,3286 @@
+  model Z3Shortname {
+    value: shortname;
+  }
+
+  /** zone 3 label (first part) */
+  @inherit(r_8, w_8)
+  @ext(0x17, 0)
+  model Z3Name1 {
+    value: zname;
+  }
+
+  /** zone 3 label (second part) */
+  @inherit(r_8, w_8)
+  @ext(0x18, 0)
+  model Z3Name2 {
+    value: zname;
+  }
+
+  /** zone 3 status */
+  @inherit(r_8)
+  @ext(0x1b, 0)
+  model Z3Status {
+    value: zonestatus3;
+  }
+
+  /** quick veto end time zone 3 */
+  @inherit(r_8)
+  @ext(0x1e, 0)
+  @condition(Basv3.Hc3CircuitType.value, "1")
+  model Z3QuickVetoEndTime {
+    value: HTI;
+  }
+
+  /** end time of holidays for zone 3 */
+  @inherit(r_8, w_8)
+  @ext(0x20, 0)
+  @condition(Basv3.Hc3CircuitType.value, "1")
+  model Z3HolidayEndTime {
+    value: HTI;
+  }
+
+  /** start time of holidays for zone 3 */
+  @inherit(r_8, w_8)
+  @ext(0x21, 0)
+  @condition(Basv3.Hc3CircuitType.value, "1")
+  model Z3HolidayStartTime {
+    value: HTI;
+  }
+
+  /** manual heating setpoint zone 3 */
+  @inherit(r_8, w_8)
+  @ext(0x22, 0)
+  @condition(Basv3.Hc3CircuitType.value, "1")
+  model Z3HeatingSetpointTemp {
+    value: tempv;
+  }
+
+  /** manual cooling setpoint zone 3 */
+  @inherit(r_8, w_8)
+  @ext(0x23, 0)
+  @condition(Basv3.Hc3CircuitType.value, "1")
+  @condition(Basv3.Hc3CoolingEnabled.value, "1")
+  model Z3CoolingSetpointTemp {
+    value: tempv;
+  }
+
+  /** quick veto end date zone 3 */
+  @inherit(r_8)
+  @ext(0x24, 0)
+  @condition(Basv3.Hc3CircuitType.value, "1")
+  model Z3QuickVetoEndDate {
+    value: date;
+  }
+
+  /** quick veto duration zone 3 */
+  @inherit(r_8, w_8)
+  @ext(0x26, 0)
+  @condition(Basv3.Hc3CircuitType.value, "1")
+  model Z3QuickVetoDuration {
+    @unit("h")
+    value: EXP;
+  }
+
+  /** room humidity zone 3 */
+  @inherit(r_8)
+  @ext(0x28, 0)
+  model Z3RoomHumidity {
+    value: percentv;
+  }
+
+  // ##### timers #####
+
+  /** default *r */
+  model r_9 {}
+
+  @inherit(r_9)
+  @ext(0xb5, 0x55, 0xa0)
+  model UnknownValue_a0 {
+    @maxLength(8)
+    value: HEX;
+  }
+
+  // timers hot water
+
+  /** default *r */
+  @base(MF, 0x55, 0xa5, 0, 0x2)
+  model r_10 {}
+
+  /** default *w */
+  @write
+  @base(MF, 0x55, 0xa6, 0, 0x2)
+  model w_9 {}
+
+  /** hwc timer Configuration */
+  @inherit(r_9)
+  @ext(MF, 0x55, 0xa3, 0, 0x2)
+  @condition(Basv3.HwcEnabled.value, "1")
+  model HwcTimer_Config {
+    /** Configuration */
+    @maxLength(9)
+    value: HEX;
+  }
+
+  /** hwc timer amount */
+  @inherit(r_9)
+  @ext(MF, 0x55, 0xa4, 0, 0x2)
+  @condition(Basv3.HwcEnabled.value, "1")
+  model HwcTimer_TimeSlotsPerWeekday {
+    /** Amount of configured slots per weekday */
+    value: slotCountWeek;
+  }
+
+  /** hwc timer monday read */
+  @inherit(r_10)
+  @ext(0, 0)
+  @condition(Basv3.HwcEnabled.value, "1")
+  model HwcTimer_Monday {
+    value: rTimeSlotWithoutTemp;
+  }
+
+  /** hwc timer monday write */
+  @inherit(w_9)
+  @ext(0, 0)
+  @condition(Basv3.HwcEnabled.value, "1")
+  model HwcTimer_MondayWrite {
```

Comment:
I guess these have to be reworked to have read+write with the same name as desired (rather a comment to myself @john30 )